docs/command_line_reference.rst

```diff
@@ -685,6 +685,28 @@ Client certificates can be presented regardless of the ``verify_certs`` setting,
 * Enable SSL, verify server certificates using private CA: ``--client-options="use_ssl:true,verify_certs:true,ca_certs:'/path/to/cacert.pem'"``
 * Enable SSL, verify server certificates using private CA, present client certificates: ``--client-options="use_ssl:true,verify_certs:true,ca_certs:'/path/to/cacert.pem',client_cert:'/path/to/client_cert.pem',client_key:'/path/to/client_key.pem'"``
 
+**Converting PKCS#12 Files to PEM Format**
+
+Rally supports PEM format for CA and client certificates. Certificates in PKCS#12 formatted keystores will need to be exported to PEM format in order for Rally to use them. The ``openssl`` command can be used to export certificates from a PKCS#12 keystore. In the following example, PKCS#12 file ``elastic-stack-ca.p12`` is generated by the Elasticsearch ``elasticsearch-certutil ca`` command and contains only the CA certificate and private key.
+
+* Export CA certificate:
+
+ ::
+
+   openssl pkcs12 -in elastic-stack-ca.p12 -out cacert.pem
```

Comment:
I missed it since openssl on macOS behaves slightly differently than Linux when an export password is not supplied. Sorry for that, I've added `-nokeys` to correct it.